gqueries/output_elements/output_series/vertical_stacked_bar_number_of_households_per_space_heating_technology/number_of_residences_per_build_year/number_of_residences_1945_1964.gql

```diff
@@ -0,0 +1,8 @@
+- unit = #
+- query = 
+    SUM(
+        V(households_useful_demand_for_space_heating_apartments_1945_1964, number_of_units),
+        V(households_useful_demand_for_space_heating_detached_houses_1945_1964, number_of_units),
+        V(households_useful_demand_for_space_heating_semi_detached_houses_1945_1964, number_of_units),
+        V(households_useful_demand_for_space_heating_terraced_houses_1945_1964, number_of_units),
+    )
```

Comment:
Wouldn't using the node groups be easier? For example: `SUM(V(G(households_1945_1964),number_of_units))`